src/shogun/classifier/PluginEstimate.cpp

```diff
@@ -22,16 +22,43 @@ CPluginEstimate::CPluginEstimate(float64_t pos_pseudo, float64_t neg_pseudo)
 {
 	m_parameters->add(&m_pos_pseudo,
 			"pos_pseudo","pseudo count for positive class");
+	watch_param(
+	    "pos_pseudo", &m_pos_pseudo,
+	    AnyParameterProperties(
+	        "pseudo count for positive class", MS_NOT_AVAILABLE,
```

Comment:
@guruhegde  I have one more request to keep this cleaner:
Can you make the second two parameter of the `AnyParameterProperties` constructor have default values set to false.
And then don't put `MS_NOT_AVAILABLE` every time, but only the descriptions ...

Reply:
This way, things will look way more neat

Reply:
@karlnapf ok. That looks lot cleaner. Will modify and update.

would you like me to send a PR to set default argument in existing constructor of AnyParameterProperties? or Make new constructor(like you commented in #4132)

I guess this PR can wait till above-mentioned changes get merged.

Reply:
Do a separate PR since that is very small and minimal. Then rebase this one here